cpp/cmake/thirdparty/get_libcurl.cmake

```
@@ -22,13 +22,13 @@ function(find_and_configure_libcurl)
   endif()
 
   rapids_cpm_find(
-    CURL 7.87.0
+    CURL 8.5.0
     GLOBAL_TARGETS libcurl
     BUILD_EXPORT_SET kvikio-exports
     INSTALL_EXPORT_SET kvikio-exports
     CPM_ARGS
-    GIT_REPOSITORY https://github.com/curl/curl
-    GIT_TAG curl-7_87_0
+   GIT_REPOSITORY https://github.com/curl/curl
+    GIT_TAG curl-8_5_0
```

Looks like [the style check failed]( https://github.com/rapidsai/kvikio/actions/runs/12244907972/job/34157647803?pr=574#step:7:1596 ) due to the indent change here

```suggestion
    GIT_REPOSITORY https://github.com/curl/curl
    GIT_TAG curl-8_5_0
```